2025-08-29 14:23:13 INFO root Start primary API version 1.8.21
2025-08-29 14:23:13 INFO root loading script: main
2025-08-29 14:23:15 INFO root successfully loaded script: main
2025-08-29 14:23:51 WARNING root Bezel button info not found for: device 4-0
2025-08-29 14:23:58 INFO root Device 4-0 VTLP sync handling began
2025-08-29 14:23:58 INFO root Device 4-0 VTLP sync handling ended
2025-08-29 14:24:00 ERROR root Failed to run wait callback function: Wait.w
Traceback (most recent call last):
  File "/remotelib_qxi/__init__.py", line 280, in w
    self.receive_message(data,ipaddress)
  File "/remotelib_qxi/__init__.py", line 344, in receive_message
    WrapperBasics.check_create_objects_queue()
  File "/remotelib_qxi/__init__.py", line 268, in check_create_objects_queue
    dev = WrapperBasics.constructors[data['device type']](instance,alias,data)
  File "/remotelib_qxi/device/UIDevice.py", line 66, in __init__
    self.check_where_used()
  File "/remotelib_qxi/device/UIDevice.py", line 84, in check_where_used
    self.where_used_items[type][id] = self.item_constructors[type](self.WrapperBasics,item_alias,data,host=self)
  File "/remotelib_qxi/ui/Label.py", line 54, in __init__
    for item in self.event_attrs:
AttributeError: 'ObjectWrapper' object has no attribute 'event_attrs'
2025-08-29 14:24:14 INFO root Device 4-0 VTLP sync handling began
2025-08-29 14:24:14 INFO root Device 4-0 VTLP sync handling ended
2025-08-29 14:24:14 INFO root Device 4-0 VTLP sync handling began
2025-08-29 14:24:14 INFO root Device 4-0 VTLP sync handling ended
2025-08-29 14:24:14 ERROR root Debug Server restarted:Listening
2025-08-29 14:24:44 ERROR root Debug Server restarted:Listening
